app/(tabs)/Deposit.tsx

```diff
@@ -82,21 +84,12 @@ export default function Deposit() {
               ...styles.amountLabel,
             }}
           >
-            { i18n.t('deposit_form_amount') }
+            {i18n.t('deposit_form_amount')}
```

Comment:
I don't think this was changed on purpose? will cause formatting back and forth in future. Please undo

Reply:
strange, in my branch it appears only once...

<img width="367" alt="Screenshot 2023-11-29 at 15 35 30" src="https://github.com/LiraShapira/app/assets/64899373/9ed23ab9-a7c8-4756-a6ee-0d16b92cc5a3">

Reply:
I have automatic formatting so It just changes it like this every time I save(I just checked as well) It strange that it appears like a change on git

same for the other formatting issues
would you like me to cancel it? It just usually makes life easier..